base/cvd/cuttlefish/host/commands/cvd/instances/instance_group_record.cpp

```diff
@@ -151,6 +179,19 @@ std::string LocalInstanceGroup::MetricsDir() const {
   return HomeDir() + "/metrics";
 }
 
+std::string LocalInstanceGroup::ArtifactsDir() const {
+  return BaseDir() + "/artifacts";
+}
+
+std::string LocalInstanceGroup::ProductDir(int instance_index) const {
+  return fmt::format("{}/{}", ArtifactsDir(), std::to_string(instance_index));
```

Comment:
nit: this should work without `std::to_string`.

Reply:
Done